### DISCUSSION TITLE
Norm of Burgers vector in bcc and fcc: GND calculation

### DISCUSSION THREAD

**Discussion by stargazingdays** [mtex-toolbox/mtex#702] [Ask Anything]
Hi,

I am new to GND calculation and I got a quick question after reading the tutorial page in documentation: https://mtex-toolbox.github.io/GND.html
'in bcc and fcc the norm of the burgers vector is sqrt(3)/2 * a'
Here does the norm refers to the norm of the full dislocation in these structures? In this case should the value for fcc be sqrt(2)/2*a? This is also something that requires input in OIM and ATEX and I got confusing reading posts and discussions online.
Thanks in advance!

Best,
Jiarui

**Comment by Savillian** [comment]
I also had the same question. With the burgers vector being a/2 <110> for FCC, this returns the sqrt(2)/2*a value. The comment in the GND calculation process seems to use the magnitude of the {111} plane instead. I could not reconcile this crystallographically.
